src/themes/teams/components/Dropdown/dropdownMessageLoadingStyles.ts

```diff
@@ -0,0 +1,11 @@
+import ListItem from '../../../../components/List/ListItem'
+import { DropdownMessageLoadingProps } from '../../../../components/Dropdown/DropdownMessageLoading'
+import { ComponentSlotStylesInput, ICSSInJSStyle } from '../../../types'
+
+const dropdownMessageLoadingStyles: ComponentSlotStylesInput<DropdownMessageLoadingProps, any> = {
+  root: ({ variables: v }): ICSSInJSStyle => ({
+    [`&.${ListItem.className}`]: { backgroundColor: v.listItemBackgroundColor },
```

Comment:
do we want to consider `ListItem` as a child element only? It seems that this styling is generic enough, so that client would like to see it being applied to either custom `<li>` or even `<div>` element passed as a slot value. Should we omit this selector altogether?

Reply:
Removed this selector at all